src/antares/model/binding_constraint.py

```diff
@@ -45,6 +44,16 @@ class TermOperators(BaseModel):
     weight: Optional[float] = None
     offset: Optional[int] = None
 
+    @computed_field  # type: ignore[misc]
+    @property
+    def weight_offset(self) -> str:
```

Comment:
I don't understand why you've introduced that

Reply:
This is for the way the weight and offset are written in the .ini

Reply:
Ok I see. IMO you should make it a method and not a property as it's confusing.